### DISCUSSION TITLE
Issue with Astroid One template update on Joomla core menu module

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#639] [Q&A]
Hi,
in footer navigation I use Joomla core menu modules. Since the last update of One Template (3.0.11 -> 3.0.12), links are no longer wrapped in `<span>` tag but directly in `<a>` tag, which means that my classes are no longer taken into account. 
I guess that since these modules are part of the Joomla core, they shouldn't be taken into account when the template is updated?

Thanks,
Lorenzo

**Astroid One Template 3.0.11**
![dev-progetto](https://github.com/templaza/astroid-framework/assets/566221/ce985fa8-b670-4e8b-a27b-9c324c1fe7a7)

**Astroid One Template 3.0.12**
![beta-progetto](https://github.com/templaza/astroid-framework/assets/566221/15122c20-3fed-4a88-ab6f-a0ec6a19d103)

**Comment by iorbita** [comment]
I changed the rule in my scss file, of course the problem is minimal, but I'd be curious to know if it's normal that updating the template can change the HTML structure of a Joomla core module. Thanks.

**Comment by Chacapamac** [comment]
> I changed the rule in my scss file, of course the problem is minimal, but I'd be curious to know if it's normal that updating the template can change the HTML structure of a Joomla core module. Thanks.

Yeah, that’s a bummer, You can probably override the html?

**Comment by iorbita** [comment]
... I changed scss rule, it took a couple of seconds...... but it's a shame that the core module has been overrided by the template...
